### PR TITLE
[web][SIMS #371]Student navigates in account

### PR DESCRIPTION
**Pull request by andrewsignori-aot**
- Upgraded Veutify from alpha 0 to alpha 6;
- Areas using more Veutify components
    - Institution
        - Top bar
         - Login
         - Home Side Menu (thanks @ann-aot )
         - Management Side Menu
    - Student
        - Top bar
        - Login
- Reorganized the Institution components to adapt better to the Veutify layout reducing nesting (InstitutionFullLayout.vue removed).
- Added BC Sans to be used as a global font as per described on BC Dev Hub (https://developer.gov.bc.ca/Design-System/Typography). There is a npm package, but seems that it is not using the woff2 files available.

image

image

image

image

### CODE REVIEW

File: sources/packages/web/src/App.vue [hunk 1]
```diff
@@ -1,19 +1,31 @@
...
 <script lang="ts">
 export default {};
 </script>
 
-<style lang="scss" scoped>
-.auto-pos {
-  position: auto;
+<style>
+html * {
+  font-family: "BCSans", "Noto Sans", Verdana, Arial, sans-serif, "primeicons";
```
Comment (ann-aot) [inline sources/packages/web/src/App.vue:14]: can we move these styles to a CSS file maybe like `base.css`

Comment (andrewsignori-aot) [reply sources/packages/web/src/App.vue]: Moved 👍

**Review by abschwenker — APPROVED** [review]

**Review by ann-aot — APPROVED** [review]
LGTM

**Review by guru-aot — APPROVED** [review]
LGTM

**Pull request merged**